rocolatey-cli/src/main.rs

```diff
@@ -47,6 +47,18 @@ async fn main() {
         .subcommand(
             SubCommand::with_name("outdated")
                 .about("Returns a list of outdated packages.")
+                .arg(
+                    Arg::with_name("ignore-pinned")
+                        .short("ip")
```

Comment:
I like the idea but will remove the short version of the parameter in order to stay consistent with Chocolatey